src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -624,6 +624,10 @@
     <value>[default]</value>
     <comment>This is used in the description of an action which changes the color of selected text, as a placeholder for a color, to indicate that the default (foreground or background) color will be used.</comment>
   </data>
+  <data name="HandleUri_default_action" xml:space="preserve">

```

Comment:
if we just don't generate a name for this action, it can never show up in the command palette. prefer this IMO